localizations/no.yml

```diff
@@ -1,4 +1,5 @@
 !!str no:
+# Per Loren 5/6/2015 - "no" is a reserved word in YAML (news to me), so you have to escape the first line in no.yml like this: `!!str no:`
```

Comment:
@dawnpm , this should be tweaked slightly:

1. Comments should be placed before the line(s) they are addressing;
2. We try to wrap lines around 80-90 char, as that makes it easier to read when one has several files open side-ty-side. We can also edit that a bit:
```ruby
# "no" is a reserved word in YAML, so we have to force it to be evaluated as a string:
!!str no:
```